src/OmniSharp.Bootstrap/project.json

```diff
@@ -13,19 +13,21 @@
     "omnisharp.bootstrap": "run"
   },
   "frameworks": {
-    "dnx451": {
+    "net451": {
       "frameworkAssemblies": {
         "System.Runtime": ""
       }
     },
-    "dnxcore50": {
-      "imports": "portable-net45+win8",
+    "netstandard1.5": {
+      "imports": [
+        "dotnet5.6",
+        "dnxcore50",
+        "portable-net45+win8"
+      ],
       "dependencies": {
-        "NETStandard.Library": "1.0.0-rc2-23910",
-        "System.Dynamic.Runtime": "4.0.11-rc2-23911",
-        "System.Linq.Expressions": "4.0.11-rc2-23911",
-        "System.Security.Cryptography.Algorithms": "4.0.0-rc2-23907",
-        "System.Xml.XDocument": "4.0.11-rc2-23911"
+        "NETStandard.Library": "1.5.0-*",
+        "System.Dynamic.Runtime": "4.0.11-*",
+        "System.Security.Cryptography.Algorithms": "4.1.0-*"
```

Comment:
This version need to be pinned down. There is flawed dependency introduced by other sources.